### DISCUSSION TITLE
MEO satellite: eclipse season & earth’s shadow

### DISCUSSION THREAD

**Discussion by coandrei** [skyfielders/python-skyfield#576] [Q&A]
I am wondering if this library allows to determine when a MEO satellite orbiting the earth is in the eclipse season and in the Earth’s shadow. And if yes, how these periods impacts the satellite performance?

Here are some of the current results: [Galileo L10](https://doi.org/10.3390/s21051695)

Open for collaboration and producing a further refined analysis.

**Comment by brandon-rhodes** [comment]
I think I remember seeing that kind of analysis done with Skyfield. Try searching the closed Issues — here are a few that might be relevant:

https://github.com/skyfielders/python-skyfield/issues/371
https://github.com/skyfielders/python-skyfield/issues/377
https://github.com/skyfielders/python-skyfield/issues/380
https://github.com/skyfielders/python-skyfield/issues/384

(With the most recent ones perhaps being the most useful.)
